learning_journal/scripts/initializedb.py

```diff
@@ -40,6 +51,8 @@ def main(argv=sys.argv):
 
     with transaction.manager:
         dbsession = get_tm_session(session_factory, transaction.manager)
+        for entry in ENTRIES:
+            new_entry = Entry(title=entry['title'], body=entry['body'], creation_date=entry['creation_date'])
+
+        dbsession.add(new_entry)
```

Comment:
If you leave this code block here, then what's going to happen is that if you try to run `initializedb` twice it'll yell at you. Not gonna ding you on it, just want you to be aware.